hackathon_site/registration/jinja2/registration/pending_activation.html

```diff
@@ -0,0 +1,14 @@
+{% extends "event/base.html" %}
+
+{% block body %}
+<div class="container">
+    <div class="section">
+        <div class="loginDiv z-depth-3 row">
+            <div class="row col s12 flexColCenter">
+                <h1 class="formH1">Activate your account</h1>
+                <p>In order to continue with your application, we first need to verify your email. Please check your email (maybe in the junk folder) for a welcome message from <a href="mailto:{{ from_email }}">{{ from_email }}</a>, with more instructions to activate your account. Once you've completed those steps, return to the site to continue with your application!</p>
```

Comment:
Just shortening it. It was way too long and I wouldn't have read it.

You must **verify your email** before you log in. Look for an email (maybe in the junk folder) from webmaster@localhost, to activate your account in order to continue with your application!